[Fabric version 0.7.3, API version 0.33.1] Latest 1.17 snapshot Fabric API immediately crashes when starting Minecraft.

As said in the title, the game can not load and crashes. The only mod in the folder in the Fabric API and using the latest launches as of writing this (which is 0.7.3). No idea what could have caused this.

---- Minecraft Crash Report ----
// I bet Cylons wouldn't have this problem.

Time: 4/26/21 8:34 PM
Description: Initializing game

java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'fabric-lifecycle-events-v1'!
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:53)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke(EntrypointUtils.java:36)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointClient.start(EntrypointClient.java:32)
	at net.minecraft.class_310.<init>(class_310.java:442)
	at net.minecraft.client.main.Main.main(Main.java:179)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)
	Suppressed: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
		at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)
		at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:208)
		at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:178)
		at org.spongepowered.asm.mixin.transformer.FabricMixinTransformerProxy.transformClassBytes(FabricMixinTransformerProxy.java:23)
		at net.fabricmc.loader.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:157)
		at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:150)
		at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
		at java.lang.Class.getDeclaredMethods0(Native Method)
		at java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
		at java.lang.Class.getDeclaredMethods(Class.java:1975)
		at net.fabricmc.loader.util.DefaultLanguageAdapter.create(DefaultLanguageAdapter.java:64)
		at net.fabricmc.loader.EntrypointStorage$NewEntry.create(EntrypointStorage.java:112)
		at net.fabricmc.loader.EntrypointStorage$NewEntry.getOrCreate(EntrypointStorage.java:99)
		at net.fabricmc.loader.EntrypointStorage.lambda$getEntrypointContainers$1(EntrypointStorage.java:184)
		at net.fabricmc.loader.EntrypointStorage$$Lambda$2660/1483059446.get(Unknown Source)
		at net.fabricmc.loader.entrypoint.EntrypointContainerImpl.getEntrypoint(EntrypointContainerImpl.java:37)
		at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:50)
		... 11 more
	Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Callback method onUnloadWorldAtShutdown(Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;Ljava/util/Iterator;Lnet/minecraft/class_3218;)V in fabric-lifecycle-events-v1.mixins.json:MinecraftServerMixin failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap fabric-lifecycle-events-v1-refmap.json
		at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.postInject(InjectionInfo.java:408)
		at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1291)
		at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1042)
		at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:395)
		at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:320)
		at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:345)
		at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:569)
		at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:351)
		... 27 more
	Suppressed: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
		at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)
		at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:208)
		at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:178)
		at org.spongepowered.asm.mixin.transformer.FabricMixinTransformerProxy.transformClassBytes(FabricMixinTransformerProxy.java:23)
		at net.fabricmc.loader.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:157)
		at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:150)
		at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
		at net.fabricmc.fabric.api.event.lifecycle.v1.ServerLifecycleEvents.lambda$static$1(ServerLifecycleEvents.java:35)
		at net.fabricmc.fabric.api.event.lifecycle.v1.ServerLifecycleEvents$$Lambda$2682/2116343141.apply(Unknown Source)
		at net.fabricmc.fabric.impl.base.event.ArrayBackedEvent.update(ArrayBackedEvent.java:47)
		at net.fabricmc.fabric.impl.base.event.ArrayBackedEvent.<init>(ArrayBackedEvent.java:38)
		at net.fabricmc.fabric.impl.base.event.EventFactoryImpl.createArrayBacked(EventFactoryImpl.java:46)
		at net.fabricmc.fabric.impl.base.event.EventFactoryImpl.createArrayBacked(EventFactoryImpl.java:42)
		at net.fabricmc.fabric.api.event.EventFactory.createArrayBacked(EventFactory.java:57)
		at net.fabricmc.fabric.api.event.lifecycle.v1.ServerLifecycleEvents.<clinit>(ServerLifecycleEvents.java:35)
		at net.fabricmc.fabric.impl.event.lifecycle.LegacyEventInvokers.onInitialize(LegacyEventInvokers.java:31)
		at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointClient$$Lambda$2659/696659808.accept(Unknown Source)
		at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:50)
		... 11 more
	Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Callback method onUnloadWorldAtShutdown(Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;Ljava/util/Iterator;Lnet/minecraft/class_3218;)V in fabric-lifecycle-events-v1.mixins.json:MinecraftServerMixin failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap fabric-lifecycle-events-v1-refmap.json
		at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.postInject(InjectionInfo.java:408)
		at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1291)
		at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1042)
		at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:395)
		at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:320)
		at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:345)
		at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:569)
		at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:351)
		... 28 more
	Suppressed: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
		at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)
		at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:208)
		at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:178)
		at org.spongepowered.asm.mixin.transformer.FabricMixinTransformerProxy.transformClassBytes(FabricMixinTransformerProxy.java:23)
		at net.fabricmc.loader.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:157)
		at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:150)
		at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
		at net.fabricmc.fabric.api.networking.v1.S2CPlayChannelEvents.lambda$static$1(S2CPlayChannelEvents.java:36)
		at net.fabricmc.fabric.api.networking.v1.S2CPlayChannelEvents$$Lambda$2683/679359967.apply(Unknown Source)
		at net.fabricmc.fabric.impl.base.event.ArrayBackedEvent.update(ArrayBackedEvent.java:47)
		at net.fabricmc.fabric.impl.base.event.ArrayBackedEvent.<init>(ArrayBackedEvent.java:38)
		at net.fabricmc.fabric.impl.base.event.EventFactoryImpl.createArrayBacked(EventFactoryImpl.java:46)
		at net.fabricmc.fabric.impl.base.event.EventFactoryImpl.createArrayBacked(EventFactoryImpl.java:42)
		at net.fabricmc.fabric.api.event.EventFactory.createArrayBacked(EventFactory.java:57)
		at net.fabricmc.fabric.api.networking.v1.S2CPlayChannelEvents.<clinit>(S2CPlayChannelEvents.java:36)
		at net.fabricmc.fabric.impl.networking.OldNetworkingHooks.onInitialize(OldNetworkingHooks.java:27)
		at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointClient$$Lambda$2659/696659808.accept(Unknown Source)
		at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:50)
		... 11 more
	Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Callback method onUnloadWorldAtShutdown(Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;Ljava/util/Iterator;Lnet/minecraft/class_3218;)V in fabric-lifecycle-events-v1.mixins.json:MinecraftServerMixin failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap fabric-lifecycle-events-v1-refmap.json
		at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.postInject(InjectionInfo.java:408)
		at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1291)
		at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1042)
		at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:395)
		at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:320)
		at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:345)
		at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:569)
		at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:351)
		... 28 more
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:208)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:178)
	at org.spongepowered.asm.mixin.transformer.FabricMixinTransformerProxy.transformClassBytes(FabricMixinTransformerProxy.java:23)
	at net.fabricmc.loader.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:157)
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:150)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at net.fabricmc.fabric.api.event.lifecycle.v1.ServerWorldEvents.lambda$static$1(ServerWorldEvents.java:32)
	at net.fabricmc.fabric.api.event.lifecycle.v1.ServerWorldEvents$$Lambda$2681/177861902.apply(Unknown Source)
	at net.fabricmc.fabric.impl.base.event.ArrayBackedEvent.update(ArrayBackedEvent.java:47)
	at net.fabricmc.fabric.impl.base.event.ArrayBackedEvent.<init>(ArrayBackedEvent.java:38)
	at net.fabricmc.fabric.impl.base.event.EventFactoryImpl.createArrayBacked(EventFactoryImpl.java:46)
	at net.fabricmc.fabric.impl.base.event.EventFactoryImpl.createArrayBacked(EventFactoryImpl.java:42)
	at net.fabricmc.fabric.api.event.EventFactory.createArrayBacked(EventFactory.java:57)
	at net.fabricmc.fabric.api.event.lifecycle.v1.ServerWorldEvents.<clinit>(ServerWorldEvents.java:32)
	at net.fabricmc.fabric.impl.event.lifecycle.LifecycleEventsImpl.onInitialize(LifecycleEventsImpl.java:50)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointClient$$Lambda$2659/696659808.accept(Unknown Source)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:50)
	... 11 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Callback method onUnloadWorldAtShutdown(Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;Ljava/util/Iterator;Lnet/minecraft/class_3218;)V in fabric-lifecycle-events-v1.mixins.json:MinecraftServerMixin failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap fabric-lifecycle-events-v1-refmap.json
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.postInject(InjectionInfo.java:408)
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1291)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1042)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:395)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:320)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:345)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:569)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:351)
	... 28 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:53)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke(EntrypointUtils.java:36)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointClient.start(EntrypointClient.java:32)
	at net.minecraft.class_310.<init>(class_310.java:442)

-- Initialization --
Details:
Stacktrace:
	at net.minecraft.client.main.Main.main(Main.java:179)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)

-- System Details --
Details:
	Minecraft Version: 21w16a
	Minecraft Version ID: 21w16a
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 5830361120 bytes (5560 MB) / 6543114240 bytes (6240 MB) up to 8589934592 bytes (8192 MB)
	CPUs: 8
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx8G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Fabric Mods: 
		fabric: Fabric API 0.33.1+1.17
		fabric-api-base: Fabric API Base 0.2.2+23b3c92ed1
		fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.0.2+413ddf64d1
		fabric-biome-api-v1: Fabric Biome API (v1) 3.1.8+413ddf64d1
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.5+92519afad1
		fabric-command-api-v1: Fabric Command API (v1) 1.1.1+bb687600d1
		fabric-commands-v0: Fabric Commands (v0) 0.2.2+92519afad1
		fabric-containers-v0: Fabric Containers (v0) 0.1.12+41a68552d1
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.2.2+23b3c92ed1
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.3+92519afad1
		fabric-dimensions-v1: Fabric Dimensions API (v1) 2.0.9+14efdf6ed1
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.0.3+92519afad1
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.8+d748de6cd1
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.1+92519afad1
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.6+92519afad1
		fabric-item-api-v1: Fabric Item API (v1) 1.2.4+23b3c92ed1
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.2.7+12865e78d1
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.4+524a2e85d1
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.2+36b77c3ed1
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.4.4+23b3c92ed1
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.4+92519afad1
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.3+92519afad1
		fabric-models-v0: Fabric Models (v0) 0.3.0+ecfd213cd1
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.7+413ddf64d1
		fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.11+413ddf64d1
		fabric-networking-v0: Fabric Networking (v0) 0.3.2+92519afad1
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.10.5+40e100ebd1
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.3+92519afad1
		fabric-particles-v1: Fabric Particles (v1) 0.2.4+92519afad1
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.7.8+413ddf64d1
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.4.2+12865e78d1
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.8+413ddf64d1
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.0+12865e78d1
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.5+92519afad1
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.13+92519afad1
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.2+92519afad1
		fabric-rendering-v1: Fabric Rendering (v1) 1.6.0+12865e78d1
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.4.6+12865e78d1
		fabric-screen-api-v1: Fabric Screen API (v1) 1.0.1+41a68552d1
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.8+23b3c92ed1
		fabric-structure-api-v1: Fabric Structure API (v1) 1.1.8+23b3c92ed1
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.1.3+41a68552d1
		fabric-textures-v0: Fabric Textures (v0) 1.0.6+92519afad1
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.2.10+f5d458fcd1
		fabricloader: Fabric Loader 0.11.3
		java: Java HotSpot(TM) 64-Bit Server VM 8
		minecraft: Minecraft 1.17-alpha.21.16.a
	Launched Version: fabric-loader-0.11.3-21w16a
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: NO CONTEXT
	GL Caps: Using framebuffer using OpenGL 3.2
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	CPU: <unknown>

--- a/build.gradle
+++ b/build.gradle
@@ -124,6 +124,9 @@ allprojects {
 		minecraft "com.mojang:minecraft:$Globals.mcVersion"
 		mappings "net.fabricmc:yarn:${Globals.mcVersion}${Globals.yarnVersion}:v2"
 		modCompile "net.fabricmc:fabric-loader:${Globals.loaderVersion}"
+
+		// TODO remove when updating to loom 0.7
+		annotationProcessor "net.fabricmc:fabric-mixin-compile-extensions:0.4.2"
 	}
 
 	configurations {

--- a/build.gradle
+++ b/build.gradle
@@ -126,7 +126,7 @@ allprojects {
 		modCompile "net.fabricmc:fabric-loader:${Globals.loaderVersion}"
 
 		// TODO remove when updating to loom 0.7
-		annotationProcessor "net.fabricmc:fabric-mixin-compile-extensions:0.4.2"
+		annotationProcessor "net.fabricmc:fabric-mixin-compile-extensions:0.4.3"
 	}
 
 	configurations {

